RawVideo/RawVideo/ContentView.swift

```diff
@@ -0,0 +1,1032 @@
+//  RawVideo
+//
+//  Created by Yassir Amadh Bisteni Aldana on 29/03/23.
+//
+
+import AzureCommunicationCalling
+import SwiftUI
+import AVFoundation
+
+struct ContentView : View
+{
+    // UI
+    @State private var videoDeviceInfoItemList: [VideoDeviceInfoItem] = []
+    @State private var cameraItemList: [CameraItem] = []
+    @State private var selectedVideoDeviceInfoIndex: Int = -1
+    @State private var selectedCameraIndex: Int = -1
+    @State private var outgoingVideoStreamType: VideoStreamType = VideoStreamType.virtualOutgoing
+    @State private var incomingVideoStreamType: VideoStreamType = VideoStreamType.rawIncoming
+    @FocusState private var isFocused: Bool
+    
+    // App
+    @State private var videoDeviceInfoList: [VideoDeviceInfo] = []
+    @State private var cameraList: [AVCaptureDevice] = []
+    @State private var callClient: CallClient?
+    @State private var callAgent: CallAgent?
+    @State private var call: Call?
+    @State private var deviceManager: DeviceManager?
+    @State private var localVideoStreamObserver: LocalVideoStreamObserver?
+    @State private var virtualRawOutgoingVideoStreamObserver: VirtualRawOutgoingVideoStreamObserver?
+    @State private var screenShareRawOutgoingVideoStreamObserver: ScreenShareRawOutgoingVideoStreamObserver?
+    @State private var remoteVideoStreamObserver: RemoteVideoStreamObserver?
+    @State private var rawIncomingVideoStreamObserver: RawIncomingVideoStreamObserver?
+    @State private var callObserver: CallObserver?
+    @State private var remoteParticipantObserver: RemoteParticipantObserver?
+    @State private var screenCaptureService: ScreenCaptureService?
+    @State private var cameraCaptureService: CameraCaptureService?
+    @State private var incomingVideoStream: IncomingVideoStream?
+    @State private var remoteVideoStream: RemoteVideoStream?
+    @State private var rawIncomingVideoStream: RawIncomingVideoStream?
+    @State private var outgoingVideoStream: OutgoingVideoStream?
+    @State private var localVideoStream: LocalVideoStream?
+    @State private var virtualOutgoingVideoStream: VirtualOutgoingVideoStream?
+    @State private var screenShareOutgoingVideoStream: ScreenShareOutgoingVideoStream?
+    @State private var outoingVideoStreamRenderer: VideoStreamRenderer?
+    @State private var outgoingVideoStreamRendererView: RendererView?
+    @State private var incomingVideoStreamRenderer: VideoStreamRenderer?
+    @State private var incomingVideoStreamRendererView: RendererView?
+    @State private var outgoingPixelBuffer: CVPixelBuffer?
+    @State private var incomingPixelBuffer: CVPixelBuffer?
+    @State private var w: Double = 0.0
+    @State private var h: Double = 0.0
+    @State private var framerate: Float = 30.0
+    @State private var maxWidth: Double = 1920.0;
+    @State private var maxHeight: Double = 1080.0;
+    @State private var token: String = "ACS token"
+    @State private var meetingLink: String = "Teams meeting link"
```

Comment:
We need to add support to join a Group also , i dont think all customers have access to Teams.

Reply:
i agree but that can be made on later updates, let's try to maintain the same features for all the platforms otherwise we end up on the the same problem